env/main/templates/nginx/usegalaxy.j2

```diff
@@ -141,6 +141,11 @@ server {
         alias /;
         add_header X-Frame-Options SAMEORIGIN;
         add_header X-Content-Type-Options nosniff;
+
+        # Add upstream response headers that would otherwise be omitted
+        add_header Access-Control-Allow-Origin $upstream_http_access_control_allow_origin;
+        add_header Access-Control-Allow-Methods $upstream_http_access_control_allow_methods;
+
         # CORS
         #add_header Access-Control-Allow-Origin "http://registry.intermine.org";
         if ($http_origin ~ "^https?://(covid19\.galaxyproject\.org|registry\.intermine\.org)$") {
```

Comment:
is this still necessary then?

Reply:
Probably not, but there are additional directives I'm not setting ... we can try this on test first.

Reply:
(i mean `Expose-Headers` and `Allow-Headers`)

Reply:
> we can try this on test first.

sounds good. (i've linked this thread from the related EU playbook change, so, they'll surely check here for the conclusion.)